autoload/health/phpactor.vim

```diff
@@ -19,6 +19,26 @@ function! s:check_diagnostics(diagnostics) abort
   endfor
 endfunction
 
+function s:check_prompt_from_autocommand() abort
+  call health#report_start('Project root detection')
+  let l:shmFIsPresent = (stridx(&shortmess, 'F') >= 0)
+
+  if !l:shmFIsPresent
+    call health#report_ok('`shortmess` does not contain `F`')
+    return
+  endif
+
+  call health#report_warn(printf('`shortmess` vim option contains `F` (is %s)', &shortmess))
+
+  if v:true == g:phpactorAllowInteractiveProjectResolution
+    call health#report_warn('- interactive choice of method may be unavailable.')
+  endif
+
+  if index(g:phpactorNoninteractiveProjectResolvers, 'manual') >= 0
+    call health#report_warn('- manual input was enabled but it may not work')
+  endif
+endfunction
+
```

Comment:
I think we should improve that, the goal is that it should be understandable by someone not knowing how everything works.
It's an help to show the user how the plugin will behave and hint on how to configure it if needed.

So first, I would add an information stating which kind of root detection is used.
Then you could add one function per type of detection, each one will have the responsibility to add relevant information.

For instance, we should not bother someone with the `F` being part of `shortmessage` or not if it does not impact his current configuration.

You should also use the `{advice}` parameter for `health#report_warn` and `health#report_error` in order to explain why the `F` option is problematic and only in the case of the manual detection.